### DISCUSSION TITLE
What is the most intimidating part of tscircuit right now? How can we make it easier to create electronics?

### DISCUSSION THREAD

**Discussion by seveibar** [tscircuit/tscircuit#470] [Ideas]
We want to make onboarding to tscircuit as easy as possible, what can we do to make it super easy?

**Comment by ShiboSoftwareDev** [comment]
Atm when you start with tscircuit(snippets) you don't know it's capabilities and limitations. The ai is cool but it doesn't tell you what you can or can't do. Documentation is important but we need something easier and clearer like a menu with drag and drop or just a collection of all components and modules that exist, then you either copy the element into the code or straight to pcb-viewer/schematic-viewer. I believe new user don't want to think too much, they want to go in and see what's there, if they see modules and components that look interesting/fun they'd stick around. When we have a good showcase and indicate that you can add your own modules and contribute your own components we should get more traffic.
